modules/standard/Path.chpl

```diff
@@ -225,5 +225,40 @@ proc file.realPath(): string throws {
    if err != ENOERR then try ioerror(err, "in file.getParentName");
    return ret;
  }
+ 
+/* Join and return one or more paths, putting precedent on the last absolute
+   path seen.  Return value is the concatenation of the paths with one
+   directory separator following each non-empty argument except the last.
+   Examples:
+
+   `joinPath("/foo/bar", "/baz")` will yield `"/baz"`
+
+   `joinPath("/foo", "./baz")` will yield `"/foo/./baz"`
+
+   `joinPath("/foo/", "", "./baz")` will also yield `"/foo/./baz"`
 
+   :arg paths: Any number of paths
+   :type paths: `string`
+
+   :return: The concatenation of the last absolute path with everything following
+            it, or all the paths provided if no absolute path is present
+   :rtype: `string`
+*/
+  proc joinPath(paths: string ...?n): string {
+    var result : string = paths(1); // result variable stores final answer
+   // loop to iterate over all the paths
+    for i in 2..n {
+    var temp : string = paths(i); 
+    if temp.startsWith('/') {
+    result = temp;
+    }  
+    else if result.endsWith('/') {
+    result = result + temp;
+    }
+    else {
+     result = result + "/" + temp;
+       }
+     }
```

Comment:
The indentation for the for loop is a bit wonky.  Contents of a for loop or if statement should be slightly more indented than the loop's indentation, for clarity.
```chapel
if a {
  for i in 1..10 {
    var x = i;
    if (b) {
      var y = 10;
    }
  }
}
```

Reply:
Okay Mam I will correct it .